src/Pickles/Pickles.DocumentationBuilders.Json/Mapper/ExampleToJsonExampleMapper.cs

```diff
@@ -44,7 +46,8 @@ public JsonExample Map(Example example)
             {
                 Name = example.Name,
                 Description = example.Description,
-                TableArgument = this.tableMapper.Map(example.TableArgument)
+                TableArgument = this.tableMapper.Map(example.TableArgument),
+                Tags = (example.Tags ?? new List<string>()).ToList(),
```

Comment:
Please provide unit tests to make sure the mapping is done correctly - `ExampleToJsonExampleMapperTests` is the appropriate class.